docker-compose.yml

```diff
@@ -10,7 +15,5 @@ services:
     volumes:
       - tofnd:/.tofnd
     environment:
-      # ATTENTION: use --unsafe only for testing
-      - UNSAFE=true
-      # available cmds: create (default), stored, update, import, export
-      - MNEMONIC_CMD=create 
+      # available cmds: auto, create (default), stored, update, import, export
```

Comment:
update this comment.  eg. no update, stored -> existing, default is existing not create